Source/PickerConfiguration.swift

```diff
@@ -1,6 +1,13 @@
 import UIKit
 
 public class PickerConfiguration {
```

Comment:
But you can use it like `ImagePicker.Configuration`, with module name, and also - do we have something else except `ImagePickerConfiguration`, I mean why do wee need 2 classes?